regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -58,15 +82,20 @@ var PreambleView = ChildView.extend({
     ChildView.prototype.render.apply(this, arguments);
     this.$read = this.$el.find('#preamble-read');
     this.$write = this.$el.find('#preamble-write');
-    var section = this.$read.closest('section').attr('id');
-    var docId = section.split('-')[0];
+
+    this.section = this.$read.closest('section').attr('id');
+    this.docId = this.section.split('-')[0];
+
+    this.currentSectionId = this.section;
```

Comment:
It looks like `this.section` isn't used after the constructor--we only change and read from `this.currentSectionId`. How about condensing this to one variable?